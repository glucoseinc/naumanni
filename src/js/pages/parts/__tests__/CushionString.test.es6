import React from 'react'
import {shallow} from 'enzyme'
import renderer from 'react-test-renderer'

import {CushionString} from '../UserParts'


const takeSnapshot = (component) => {
  const tree = renderer.create(component).toJSON()
  expect(tree).toMatchSnapshot()
}

describe('CushionString', () => {
  let component = <CushionString length={4} />
  let wrapper = shallow(component)

  it('has cushionString class', () => {
    expect(wrapper.find('.cushionString').length).toBe(1)
  })
  it('match snapshot', () => {
    takeSnapshot(
      <div>
        spam
        {component}
        ham
      </div>
    )
  })
})
