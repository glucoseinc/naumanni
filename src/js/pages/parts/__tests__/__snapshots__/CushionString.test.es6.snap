// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CushionString match snapshot 1`] = `
<div>
  spam
  <span
    className="cushionString"
  >
        
  </span>
  ham
</div>
`;
